src/diffusers/__init__.py

```diff
@@ -107,6 +107,7 @@
             "ModelMixin",
             "MotionAdapter",
             "MultiAdapter",
+            "MultiControlNetModel",
```

Comment:
add `MultiControlNetModel` to top-level import because every other `MultiControlNetModels` are, e.g, `SD3MultiControlNetModel`, `FluxMultiControlNetModel`